### PR TITLE
Feature profile

### PR DESCRIPTION
**Pull request by katerynablavt**
I make changes in getaway

### CODE REVIEW

**Review by davidlyj — APPROVED** [review]
Looks good. However, minor issue with ChangeProfileOutputAdapter.java. Line 3 should be camel-cased if possible instead of kebab-cased.

**Comment by xingjianll** [comment]
We cannot have so many pull requests I see that a pull request with the same name is already merged?

**Review by davidlyj — APPROVED** [review]
Looks good. However, might wanna change line 3 in ChangeProfileOutputAdapter.java to camel-case.

**Review by davidlyj — APPROVED** [review]
Looks good

**Review by as39248 — APPROVED** [review]
looks great

File: src/main/java/interface_adapters/change_profile/ChangeProfileOutputAdapter.java [hunk 1]
```diff
@@ -33,9 +33,9 @@ public void updateDescr(ChangeProfileOutputModel outputModel) {
         String content = outputModel.getProfileUID()+outputModel.getChangeStatusMessage();
 
        // adress and port from db
-        String adress = ""; //from db!!
-        int port = 0; //from db!!
-        comManager.send(adress,port,content);
+        String adress = outputModel.getUseraddress(); //from db!!
```
Comment (xingjianll) [inline src/main/java/interface_adapters/change_profile/ChangeProfileOutputAdapter.java:36]: I'm not sure if you can access the database from the Adapters layer.

**Review by xingjianll — APPROVED** [review]
Approved, but I'm not sure if outputAdapter can access the database.

**Review by davidlyj — APPROVED** [review]
I am going to approve this. However, I think that in ChangeProfileInteractor.java, since the variables address and port and reused in multiple methods, there is a Duplicate Code code smell. As a result, you might want to use the "Extract Method" technique and maybe assign the return values of these methods to 2 attribute variables so that you can replace these lines with those variables instead.

**Pull request merged**